G/GDAL/build_tarballs.jl

```diff
@@ -24,6 +24,7 @@ elif [[ "${target}" == *-linux-* ]]; then
     # Make sure GEOS is linked against libstdc++
     atomic_patch -p1 "$WORKSPACE/srcdir/patches/geos-m4-extra-libs.patch"
```

Comment:
> Ok I guess -lstdc++ was not the (only?) missing library here.

Applying the patch might help :wink: 

```suggestion
     atomic_patch -p1 "$WORKSPACE/srcdir/patches/geos-m4-extra-libs.patch"
     atomic_patch -p1 "$WORKSPACE/srcdir/patches/configure_ac_curl_libs.patch"
```

Reply:
:sweat_smile: